content/guides/building/pre-reqs.md

```diff
@@ -108,7 +108,7 @@ Note that in addition to all the platform-specific packages, you will also need
 **Basic requirements:**
 
 ```sh
-pkgman install cmd:python3 cmd:xorriso
+pkgman install cmd:python3 cmd:xorriso zstd_devel
```

Comment:
This line is only for Haiku: devel:libzstd would fit better.

Reply:
Will do